capstone/src/main/webapp/statistics_script.js

```diff
@@ -23,41 +29,79 @@ async function getSurveyResponses() {
     '65+': 0,
     'Unknown': 0,
   };
+
+  const directExpCount = {
+    'Prefer not to answer': 0,
+    'Yes': 0,
+    'No': 0,
+  };
+
+  const completionCount = {
+    'Partial': 0,
+    'Complete': 0,
+  };
+
+  const responseTimeAverage = {
+    'Response One': 0,
+    'Response Two': 0,
+    'Response Three': 0,
+  };
+
+  /* Stores a respective field in each object of the list */
+  const scores = [];
   const genders = [];
-  const responseTimes = [];
+  const responseTimesThree = [];
   const ages = [];
   const days = [];
 
   for (let i = 0; i < list.length; i++) {
     scores.push(list[i].score);
-    responseTimes.push(list[i].responseTime);
+    responseTimesThree.push(list[i].responseTimeThree);
```

Comment:
What does the three refer to here in responseTimeThree?

Reply:
It's the third response, there are three responses per survey. The reason why I made a separate list for the third responses is because I only need those response times for the chart and for the analysis since we are only running sentiment analysis on the third question survey question

Reply:
+1 similar to my comment above